src/App.tsx

```diff
@@ -1,47 +1,37 @@
-import React, { useEffect, useState } from 'react';
+import React, { useEffect } from 'react';
```

Comment:
Consider removing the unused import of 'useEffect' from React as it is not being utilized in this file. This will help in keeping the code clean and optimized.